RecoTracker/LSTCore/src/alpaka/Event.h

```diff
@@ -18,15 +17,16 @@
 #include "HeterogeneousCore/AlpakaInterface/interface/host.h"
 
 namespace SDL {
-  template <typename TAcc>
-  class Event {};
+
+  template <typename TDev>
+  class Event;
 
   template <>
-  class Event<SDL::Acc> {
+  class Event<ALPAKA_ACCELERATOR_NAMESPACE::Device> {
```

Comment:
I thought Accelerator was more appropriate to distinguish the execution-related code, while Device is more relevant for a memory representation.
Is a CPU with different kinds of threading the same kind of device but different accelerator? or am I misinterpreting what's in https://alpaka.readthedocs.io/en/latest/basic/library.html#structure

Reply:
You're right, I'll fix it